### PR TITLE
raidboss: eureka pyros skoll triggers

### PR DESCRIPTION
**Pull request by Jaehyuk-Lee**

### CODE REVIEW

File: ui/raidboss/data/04-sb/eureka/eureka_pyros.js [hunk 1]
```diff
@@ -8,6 +8,24 @@
   },
   resetWhenOutOfCombat: false,
   triggers: [
+    {
+      id: 'Eureka Pyros Skoll Hoarhound Halo',
```
Comment (quisquous) [inline ui/raidboss/data/04-sb/eureka/eureka_pyros.js:12]: I forget what these abilities are.  Are these get in or get out? Are there better instructions than just the ability names?

Comment (Jaehyuk-Lee) [reply ui/raidboss/data/04-sb/eureka/eureka_pyros.js:12]: those are
avoid boss back
avoid boss face

I don't know natural explanation of this in English.

Comment (quisquous) [reply ui/raidboss/data/04-sb/eureka/eureka_pyros.js:12]: What about:
avoid boss back = `Responses.goFrontOrSides()`
avoid boss face = `Responses.getBehind()`

(the fact that I don't know what these are probably explains why I always got killed by Skoll :joy:)

Comment (Jaehyuk-Lee) [reply ui/raidboss/data/04-sb/eureka/eureka_pyros.js:12]: Ah, I forgot about it, reponses.

Comment (Jaehyuk-Lee) [reply ui/raidboss/data/04-sb/eureka/eureka_pyros.js:12]: Oh, there's no `awayFromBack()` or `goFrontOrSides()`.
Need new function.

Comment (Jaehyuk-Lee) [reply ui/raidboss/data/04-sb/eureka/eureka_pyros.js:12]: oh, you made new function on master branch.

**Pull request merged**